stories/assets/scss/_mixins.scss

```diff
@@ -493,4 +493,39 @@
   }
 }
 
-/* placehoder end */
\ No newline at end of file
+%header-icon{
+  @include devicebreak(mediumonlytab) {
+    margin-left: $spacing-12;
+    display: none;
+    font-size: 0;
+    background-image: none;
+    width:1.375rem;
+    height: 1.375rem;
+    &:hover{
+      animation: none;
+    }
+    &.show{
+      display: inline-block;
+    }
+  }
+}
+
+%header-close{
```

Comment:
@Ashish-srij We can use just one place holder and based on the state (open) change just the background image.

Reply:
@shishirphadke Done.